packages/frontend/src/components/pagination/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { Dispatch, SetStateAction } from "react";
+import { PaginationBall } from "./pagination-ball";
+import { ReactComponent as ArrowIcon } from "../../assets/arrow.svg";
+import { cva } from "class-variance-authority";
+
+const createArrayBetweenRange = (start: number, end: number) =>
+    Array.from({ length: end - start }, (_, i) => start + i);
+
+const LOT_OF_PAGES = 5;
```

Comment:
This could be renamed to something like `VISIBLE_PAGE_NAVIGATION_THRESHOLD`.

Reply:
I do find it more difficult to understand than a "LOT_OF_PAGES" 😅

Reply:
Maybe `VISIBLE_PAGES_THRESHOLD`, I'd remove navigation.

Reply:
I fixed to VISIBLE_PAGES_THRESHOLD. thanks